gems/pending/util/MiqSockUtil.rb

```diff
@@ -8,6 +8,10 @@ def self.getHostName
   # Return the cannonical name for host in DNS
   def self.getFullyQualifiedDomainName
     Socket.gethostbyname(Socket.gethostname).first
+  rescue => e
+    $log.error "The appliance hostname could not be resolved. Please make sure to set a valid fully qualified hostname "\
+               "for SmartState analysis to work correctly."
```

Comment:
I don't think we should hardcode anything about SmartState here.  This method is used in other locations, I believe, such as in server startup.  It will be very confusing if during server startup you get a message about SmartState.

cc @gmcculloug